package.json

```diff
@@ -30,9 +30,9 @@
     "lint-fix": "yarn lint-js -- --fix",
     "mochi":
       "mochii --mc ./firefox --default-test-path devtools/client/debugger/new",
-    "mochid": "yarn mochi -- --jsdebugger",
-    "mochir": "yarn mochi -- --repeat 10",
-    "mochih": "yarn mochi -- --setenv MOZ_HEADLESS=1",
+    "mochid": "yarn mochi -- --jsdebugger --",
```

Comment:
why do we need the two dashes at the end?

Reply:
i'll move this to a separate PR, but the dashes keep the file passed in from being in quotes.